.github/CODEOWNERS

```diff
@@ -1,10 +1,10 @@
 # CODEOWNERS info: https://help.github.com/en/articles/about-code-owners
 # Owners are automatically requested for review for PRs that changes code
 # that they own.
-* @manishrjain @vvbalaji-dgraph
-/posting/ @manishrjain @martinmr
-/ee/acl/ @manishrjain @gitlw
-/wiki/ @danielmai @MichaelJCompton
-/contrib/config/ @danielmai
-/query/ @pawanrawal
-/graphql/ @pawanrawal @MichaelJCompton
+* @akon-dey @nosql22
+/posting/ @akon-dey
+/ee/ @akon-dey
+/wiki/ @MichelDiz
```

Comment:
Wiki is dead. It was moved.

Reply:
Shall I remove line#7 @MichelDiz ?

Reply:
I guess tho. No one will work there.

Reply:
I did re-check, so it's moved to `dgraph-io/docs` & it's marked deprecated. Made the correction here https://github.com/dgraph-io/dgraph/pull/8159/commits/6a4c697fb0420c2d2e4db527e17716c557dc52fe. Going to resolve this.